CondCore/Utilities/bin/conddb_test_gt_perf.cpp

```diff
@@ -325,7 +325,7 @@ cond::TestGTPerf::TestGTPerf() : Utilities("conddb_test_gt_load") {
 // thread helpers
 
 // global counter for dummy thread measurements:
-volatile int fooGlobal = 0;
+int fooGlobal = 0;
```

Comment:
I suspect this should be ```std::atomic<int>```

Reply:
Thanks Vincenzo! I'll go with that then.